packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -510,18 +509,21 @@ export class AtomicSearchBox {
 
   private updateSuggestionElements(query: string) {
     if (!this.isPanelInFocus(this.leftPanelRef, query)) {
-      this.leftSuggestionElements = this.getSuggestionElements(
-        this.leftSuggestions
-      );
-    }
+      this.leftSuggestionElements = this.getAndFilterLeftSuggestionElements();
```

Comment:
I feel like it would make more sense to filter both sides than not 🤔

Reply:
What would you filter on ?